src/main/daml/Daml/Finance/Interface/Types/Common/Types.daml

```diff
@@ -32,19 +32,28 @@ data AccountKey = AccountKey
       -- ^ Unique identifier for an account.
   deriving (Eq, Ord, Show)
 
--- | A unique key for Instruments.
+-- | A key for Instruments.
 data InstrumentKey = InstrumentKey
   with
     depository : Party
       -- ^ Party providing depository services.
     issuer : Party
       -- ^ Issuer of instrument.
     id : Id
-      -- ^ A unique identifier for an instrument.
+      -- ^ An identifier for an instrument.
     version : Text
       -- ^ A textual instrument version.
   deriving (Eq, Ord, Show)
 
+-- | Key type of the `TimeObservable`.
+data TimeObservableKey = TimeObservableKey
```

Comment:
A similar question as with `BatchKey`. The `TimeObservableKey` is implementation-specific to the `DateClock`. Can it live in `Daml.Finance.Data`?

Reply:
Same reason as for the `BatchKey`. Currently it doesn't seem to be used  as well.